src/AccessibilityInsights.SharedUx/Controls/TestTabs/AutomatedChecksControl.xaml

```diff
@@ -78,7 +78,7 @@
                                                 AutomationProperties.Name="{x:Static Properties:Resources.btnExpandAllAutomationPropertiesName}"
                                                 AutomationProperties.AutomationId="{x:Static Properties:AutomationIDs.AutomatedChecksExpandAllButton}">
                                             <Grid>
-                                                <AccessText Text="_c" Opacity="0"/>
+                                                <AccessText Text="{x:Static Properties:Resources.AutomatedChecksControl_ExpandCollapseAll_AccessText}" Opacity="0"/>
```

Comment:
As mentioned in the comments, this intentionally changes from _c to _e to avoid conflict